packages/table/src/Row/InternalRowWithoutRT.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement, ReactNode } from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+
+import { Cell } from '../Cell';
+import { useTableContext } from '../TableContext/TableContext';
+
+import InternalRowBase from './InternalRowBase';
+import { zebraStyles } from './Row.styles';
+import { InternalRowBaseProps } from './Row.types';
+
+/**
+ * Renders basic array row data
+ */
+const InternalRowWithoutRT = ({
+  children,
+  className,
+  ...rest
+}: InternalRowBaseProps) => {
+  const { shouldAlternateRowColor } = useTableContext();
+  const { theme } = useDarkMode();
+  return (
+    <InternalRowBase
+      className={cx(
+        {
+          [zebraStyles[theme]]: shouldAlternateRowColor,
+        },
+        className,
+      )}
+      {...rest}
+    >
+      {React.Children.map(children, (child: ReactNode, index: number) => {
+        return <Cell {...(child as ReactElement).props} cellIndex={index} />;
```

Comment:
This pattern is confusing to me: Aren't rows already rendering cells?

Reply:
This was put in place because the behavior where `Cell`s inherited the `HeaderCell`s' `align` prop depended on a `cellIndex`, but there might actually be a workaround since we changed how that prop is inherited recently. I'll look into this

Reply:
Looked into it - it's going to be a bit tricky and probably not something we can change quickly now. RT doesn't expose API to know the rendered order of a cell/column because it supports reordering of columns using an array of columns. So to get the rendered order of a cell, it would have to do a search for a nested field in an array of columns for each cell, which is a pretty expensive calculation.